man/coredumpctl.xml

```diff
@@ -227,11 +235,21 @@
         <option>--output=</option>. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>debug</command></term>
+
+        <listitem><para>Invoke the a debugger on the last core dump
+        matching specified characteristics. The debugger used (either gdb or lldb)
+        will first be read from <option>--debugger</option>, then from
+        <varname>$SYSTEMD_DEBUGGER</varname>. If neither is given, then the
+        default of gdb will be used.</para></listitem>
```

Comment:
This phrasing is akward. Maybe:
```xml
By default <citerefentry><refentrytitle>gdb</refentrytitle><manvolnum>1</manvolnum></citerefentry>
will be used. This may be changed using the <option>--debugger=</option> option or the 
<varname>$SYSTEMD_DEBUGGER</varname> environment variable.
```